ttnn/cpp/ttnn/operations/pool/maxpool/max_pool2d_pybind.hpp

```diff
@@ -0,0 +1,81 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <pybind11/pybind11.h>
```

Comment:
https://github.com/tenstorrent/tt-metal/pull/10825/files#r1697623357

Reply:
can you please check ccl/all_gather binding header? it shows how to avoid these heavy includes